Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM node:lts-alpine
+FROM node:lts-slim
 
-RUN apk add --no-cache git
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get -y install git
```

Comment:
Completely optional: I think there's also a command you can run to cleanup things a bit to reduce the size of the image slightly: https://github.com/exercism/prolog-test-runner/blob/main/Dockerfile#L8